src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -525,24 +525,22 @@ private bool ValidateNonce(ContractEntry entry, ExecutionEnvironment env, ITxTra
             {
                 return true;
             }
-            
-            TraceLogInvalidTx(entry, $"INSUFFICIENT_SENDER_BALANCE: ({env.Sender})_BALANCE = {senderBalance.ToString()}");
+
+            TraceLogInvalidTx(entry,
+                $"INSUFFICIENT_SENDER_BALANCE: ({env.Sender})_BALANCE = {senderBalance.ToString()}");
             QuickFail(entry, env, txTracer);
             return false;
         }
 
         /// <summary>
-        /// Prepares and validates the account address for contract deployment.
+        ///     Prepares and validates the account address for contract deployment.
         /// </summary>
         /// <param name="recipient">Contract address.</param>
         /// <exception cref="TransactionCollisionException">Thrown when the address is already in use</exception>
         private void PrepareContractAccount(Address recipient)
         {
-            if (!_stateProvider.AccountExists(recipient))
-            {
-                return;
-            }
-            
+            if (!_stateProvider.AccountExists(recipient)) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4658486)